examples/README.md

```diff
@@ -0,0 +1,109 @@
+
+product_code:
+    TBD
+
+query_interval:
+    Time unit how often the csp-billng-adapter collectd the data from the application.
+
+reporting_api_is_cumulative:
+    Specifies if the CSP will accumulate reported values. While all CSPs will accumulate the values reported, it may not be possible to report 0. For example, in a monthly billing scenario, the csp-billing-adapter may have to report on a hourly basis a number that is not 0 such that the CSP API accumulates the values. A true setting would direct the adapater to normalize the unit count based on some to be determined normalization algorithm.
+
+reporting_interval: 
+    Sets the time when csp-billing-adapter reports to the CSP API. The values the csp-billing-adapter reports are determined by a combination of the settings for `reporting_api_is_cumulative` and `billing_interval`.
+
+support_info_collection_cmd:
+    A command executed by the adapter to collect specific information that will be useful for support situations.
+
+usage_metrics: 
+
+  {metric name}:
+    The name of the metric, for example managed_node_count
+
+    consumption_reporting:
+        Sets the way the csp-billing-adapter reports the value(s) it receives, either volume or tiered.
+
+        The volume value triggers the code to search for the highest possible dimension to use and all other configured dimensions will be reported as 0, or not reported, depending on CSP API requirements. For example:
+        
+            - The configuration is set to volume for the consumption_reporting
+            - The dimensions configuration contains 3 dimensions with a maximum setting of 30, 40, and 55
+            
+            If the value reported by the application API is 150:
+            
+                The csp-billing-adapter will report 0 for dimension 1 and 2 and report 150 for dimension 3.
+            
+            If the application API reports 36:
+
+                The csp-billing-adapter will report 0 for dimension 1, 36 for dimension 2, and 0 for dimension 3.
+            
+            In this case the application API may report only 1 dimension. If the application API reports more than 1 dimension it is an error condition.
+
+        The tiered value triggers the code to report per dimension. This has 2 implied reporting mechanisms.
+
+            One use case is the application API reports dimensions that match the configuration.
+
+                In this case, the csp-billing-adapter reports the count per dimension as obtained by the application API.
+
+            The second use case is for the application API to return only 1 value
+
+                In this case the code will divide the reported value into categories reporting for each dimension the maximum value possible. For example:
+                
+                - There are 3 dimensions with a maximum setting of 30, 40, and 55
+                - A minimum setting of 5, 31, and 41
```

Comment:
s/5/0/ after meeting on April 13 where it was decided to not include the minimum consumption setting in "min" but split it out as "min_consumption" the value 5 which indicated the minimum consumption needs to be changed to 0 in the example. The first dimension will always start at 0